rasa/telemetry.py

```diff
@@ -608,6 +608,28 @@ def initialize_error_reporting() -> None:
     from sentry_sdk.integrations.dedupe import DedupeIntegration
     from sentry_sdk.integrations.excepthook import ExcepthookIntegration
 
+    ignored_errors = [
+        KeyboardInterrupt,
+        RasaException,
+        NotImplementedError,
+        asyncio.CancelledError,
+        MemoryError,
+    ]
+
+    try:
+        from pymongo.errors import PyMongoError
+
+        ignored_errors.append(PyMongoError)
+    except ImportError:
+        pass
+
+    try:
+        from psycopg2 import OperationalError
+
+        ignored_errors.append(OperationalError)
+    except ImportError:
+        pass
```

Comment:
it might be better to define these closer to the classes that rely on pymongo / postgres 🤔